src/ports/postgres/modules/utilities/utilities.py_in

```diff
@@ -407,6 +407,15 @@ def set_client_min_messages(new_level):
     return old_msg_level
 # -------------------------------------------------------------------------
 
+def is_less_than_pg11(schema_madlib, **kwargs):
```

Comment:
I noticed that we have another function called `is_less_than_pg90` which has the same code as this function. We probably don't need the pg90 function anymore but it would be nice if the `is_less_than_pg11`  function can be generalized to take in a major postgres version and then compare the current version with the passed major version.